watchtower-plugin/src/wt_client.rs

```diff
@@ -310,11 +336,17 @@ mod tests {
     use tempdir::TempDir;
     use tokio::sync::mpsc::unbounded_channel;
 
+    #[cfg(feature = "accountable")]
     use teos_common::test_utils::{
         generate_random_appointment, get_random_appointment_receipt,
         get_random_registration_receipt, get_random_user_id,
         get_registration_receipt_from_previous,
     };
+    #[cfg(not(feature = "accountable"))]
+    use teos_common::test_utils::{
+        generate_random_appointment, get_random_registration_receipt, get_random_user_id,
+        get_registration_receipt_from_previous,
```

Comment:
This can be simplified

Reply:
I think you want the code to be like this:
` use teos_common::test_utils::{
        generate_random_appointment, #[cfg(feature = "accountable")] get_random_appointment_receipt,
        get_random_registration_receipt, get_random_user_id,
        get_registration_receipt_from_previous,
    };`
    but this is not the proper way to do it, this gives syntax error, hence I created separate functions at few places

Reply:
```
#[cfg(feature = "accountable")]
use teos_common::test_utils::get_random_appointment_receipt;
use teos_common::test_utils::{ generate_random_appointment, get_random_registration_receipt, get_random_user_id, get_registration_receipt_from_previous];
```